ironweb.d.ts

```diff
@@ -268,6 +273,6 @@ export const user: User;
 export const group: Group;
 
 export function initialize(jwtCallback: JWTCallback, passcodeCallback: PasscodeCallback): Promise<SDKInitializationResult>;
-export function createNewUser(jwtCallback: JWTCallback, passcode: string): Promise<UserCreateResponse>;
+export function createNewUser(jwtCallback: JWTCallback, passcode: string, options: UserCreateOptions): Promise<UserCreateResponse>;
```

Comment:
This entire `options` argument should be optional. As it is now, users would have to pass in a value for this, which we don't want them to have to do.